app/main.py

```diff
@@ -1 +1,12 @@
-# write your code here
+def copy_file(command: str) -> None:
+
+    if len(command.split()) != 3:
+        return
+
+    cp, file, file_copy = command.split()
```

Comment:
Don't split string twice. Create a separate variable for storing the result of split and use it where you need.